### DISCUSSION TITLE
Domyos Intense Run

### DISCUSSION THREAD

**Discussion by grigory707** [cagnulein/qdomyos-zwift#134] [Q&A] [answered]
Hello Roberto.

I can not find information, is it possible to edit preset workouts (or add my own) in Domyos INTENSE RUN treadmill?

**Comment by cagnulein** [comment, marked as answer]
Yes it's possible creating an xml file with your workout. You can find some
examples here in the repository

Il dom 24 gen 2021, 19:45 grigory707 <notifications@github.com> ha scritto:

**Reply by grigory707** [reply]
Find it! Thanks!
